package.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "@types/mediawiki",
+	"version": "0.1.0",
+	"description": "TypeScript definitions of MediaWiki interfaces",
+	"scripts": {
+		"build:types": "tsc -p tsconfig.json",
```

Comment:
I think `-p tsconfig.json` can be omitted. `tsc` uses tsconfig.json by default AFAIK.